src/main/java/com/google/codeu/data/User.java

```diff
@@ -17,4 +25,20 @@ public String getEmail(){
   public String getAboutMe() {
     return aboutMe;
   }
+  
```

Comment:
It might be a good idea to add some setters here as well, that way information can be edited.